### PR TITLE
add DocStrings to adapter.py

### PR DESCRIPTION
**Pull request by svamaa**

### CODE REVIEW

File: pyramid_oereb/lib/adapter.py [hunk 1]
```diff
@@ -55,7 +55,7 @@ def get_session(self, key, request=None):
             key (str): The key to identify the desired connection in the pool of available
                 connections.
             request (pyramid.request.Request or None): The request of the underlying pyramid
-                application. This can be useful to handle errorcases and treat sessions in
+                application. This can be useful to handle errorcases and threat sessions in
```
Comment (voisardf) [inline pyramid_oereb/lib/adapter.py:58]: threat should probably be 'to treat'

**Review by voisardf — APPROVED** [review]
thanks

File: pyramid_oereb/core/adapter.py [hunk 2]
```diff
@@ -76,16 +76,39 @@ def get_session(self, key, request=None):
 class FileAdapter(object):
 
     def __init__(self, cwd=None):
+        """
+        This Class provides a list of methods for file handling.
+
+        It uses the current working directory as central point for file handling.
```
Comment (vvmruder) [inline pyramid_oereb/core/adapter.py:82]: Maybe also add doc for paramteter?

File: pyramid_oereb/core/adapter.py [hunk 3]
```diff
@@ -76,16 +76,39 @@ def get_session(self, key, request=None):
...
+        Args:
+            path(str): The path of the new working directory.
+        """
         self._cwd_ = os.path.abspath(os.path.join(self._cwd_, path))
 
     def ls(self):
+        """
+        This method lists all content of the given path.
+
+        Args:
```
Comment (vvmruder) [inline pyramid_oereb/core/adapter.py:106]: I think we can omit Args here at all

**Review by vvmruder — CHANGES_REQUESTED** [review]
Please see my comments.

**Review by vvmruder — APPROVED** [review]
Thank you! :+1:

**Pull request merged**